### ISSUE TITLE
[Remote-SSH Bug]: Can't install extension because it's not compatible with the current version of VSCode

### ISSUE THREAD

**Issue by kotborealis** [microsoft/vscode-remote-release#8837]
### Is there an existing issue for this bug?

- [X] I have searched the existing issues

### Required Troubleshooting Steps

- [X] I have followed these troubleshooting steps
- [ ] I have tried both values of the `remote.SSH.useLocalServer` setting

### Connect Locally

It connects successfully

### Expected Behavior

Some extensions, like "cpptools" and "jupyter", cannot be installed via marketplace.

### Steps To Reproduce

1. Connect to remote machine via remote-ssh extension.
1. Create a temporary profile (Ctrl+Shift+P -> Create Temporary Profile)
1. Go to extensions, find "cpptools" or "jupyter"
1. Click install
1. See error "extension is not compatible with current version of VSCode"

### Remote-SSH Log

Sadly, there's nothing interesting or related in the Remote-SSH log. Maybe I should toggle some switch to dump more logs?

### Anything else?

Thought installing extensions from the marketplace fails, I'm able to manually download the same vsix from the marketplace and it installs just fine. 
Also works fine without remote ssh.

[Related issue on cpptools repo.](https://github.com/microsoft/vscode-cpptools/issues/11307)

**Comment by kotborealis** [comment]
Another piece of info:
I've first encountered this a few weeks ago, while setting up two completely new machines - remote host (ubuntu 20) + local host (Windows 10, new installation of vscode from the official website).

**Comment by real01bit** [comment]
I have the same problem, too. :(

**Comment by real01bit** [comment]
I tried installing each of these four extensions, and all but Remote-Tunnels installed successfully.

**Comment by kotborealis** [comment]
I'm not able to reproduce this issue anymore 😞  Not sure what has changed, but I'll post the logs in case I'll encounter it again.

**Comment by sandy081** [comment]
Makes sense. Closing the issue for now. Please reopen it if you reproduce it again.
